config/initializers/new_framework_defaults.rb

```diff
@@ -18,7 +18,7 @@
 Rails.application.config.active_record.belongs_to_required_by_default = true
 
 # Do not halt callback chains when a callback returns false. Previous versions had true.
-ActiveSupport.halt_callback_chains_on_return_false = false
+# ActiveSupport.halt_callback_chains_on_return_false = false
```

Comment:
Making a note here, supposedly this has become default behavior in rails 5, so rails 5.2 removed the option. Nothing should have changed by removing this.

Reply:
Yep. This actually moved out of active support, so it exists elsewhere. So I commented it out to keep it around just in case something weird happens, but it should exist elsewhere still.